app/views/results/edit.html.erb

```diff
@@ -76,6 +76,9 @@
       if @current_user.admin?
         num_marked = @assignment.get_num_marked
         num_assigned = @assignment.get_num_assigned
+      elsif @current_user.is_reviewer_for?(@assignment, @result)
+        num_marked = @assignment.get_num_marked(nil, @current_user.grouping_for(@assignment.id))
```

Comment:
Don't change the existing method here. Create new methods, and put them under peer reviews somewhere to keep them separate.